editor/src/clj/dev.clj

```diff
@@ -164,7 +164,9 @@
 (def curve-view (partial view-of-type curve-view/CurveView))
 
 (defn console-view []
-  (-> (view-of-type console/ConsoleNode) (g/targets-of :lines) ffirst))
+  (some-> (view-of-type console/ConsoleNode)
+          (g/targets-of :lines)
+          (ffirst)))
```

Comment:
Unrelated bug fix: Would throw instead of returning `nil` if there was no console view.